cluster/gce/trusty/configure-helper.sh

```diff
@@ -140,16 +140,13 @@ install_additional_packages() {
 # variable that will be used as the kubelet command line flags
 #   KUBELET_CMD_FLAGS
 assemble_kubelet_flags() {
-  KUBELET_CMD_FLAGS="--v=2"
+  KUBELET_CMD_FLAGS="--v=2 ${KUBELET_TEST_ARGS:-}"
   if [ -n "${KUBELET_TEST_LOG_LEVEL:-}" ]; then
-    KUBELET_CMD_FLAGS="${KUBELET_TEST_LOG_LEVEL}"
+    KUBELET_CMD_FLAGS="${KUBELET_TEST_LOG_LEVEL} ${KUBELET_TEST_ARGS:-}"
```

Comment:
Not exactly. If the test log level is set, KUBELET_CMD_FLAGS is reset to "${KUBELET_TEST_LOG_LEVEL} ${KUBELET_TEST_ARGS:-}". I guess the style is not good for read. Maybe better to write it in another way. Will update it.